src/FsAutoComplete/FsAutoComplete.Lsp.fs

```diff
@@ -121,9 +125,9 @@ type FsharpLspServer(commands: Commands, lspClient: FSharpLspClient) =
                         do! (commands.Parse filePath content version (Some tfmConfig) |> Async.Ignore)
 
                         if config.Linter then do! (commands.Lint filePath |> Async.Ignore)
-                        if config.UnusedOpensAnalyzer then  Async.Start (commands.GetUnusedOpens filePath |> Async.Ignore)
-                        if config.UnusedDeclarationsAnalyzer then Async.Start (async { ignore (commands.GetUnusedDeclarations filePath) }) //fire and forget this analyzer now that it's syncronous
-                        if config.SimplifyNameAnalyzer then Async.Start (commands.GetSimplifiedNames filePath |> Async.Ignore)
+                        if config.UnusedOpensAnalyzer then  Async.Start (commands.CheckUnusedOpens filePath)
```

Comment:
because of the other changes mentioned, a lot of ceremony here melts away, leaving just `Async.Start`